L30-cpp-server-load-reporting.md

```diff
@@ -0,0 +1,42 @@
+﻿C++ API changes for Server Load Reporting
+----
+* Author(s): AspirinSJL
+* Approver: markdroth, vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2018-07-13
+* Discussion at: TBA
```

Comment:
Please link in a discussion. I will only add comments on the document now but we should have technical discussions on grpc-io related to this proposal.